meetings/2017-01-11.md

```diff
@@ -0,0 +1,338 @@
+note that in the thread.
+
+---
+
+### meta: deprecation policy [node#7964](https://github.com/nodejs/node/pull/7964)
+
+Old PR which James is picking back up to clarify deprecation policy. Please look
+at and weigh in.
+
+Requires CTC review because it’s a change to governance processes - discusses
+deprecation policy, semver policies, what constitutes an internal API, etc.
+
+There are some notable changes:
+Docs-only deprecations would be considered semver-minor.
+Introduces concept of “end-of-life” code for removing unused code.
+
+@jasnell would prefer a vote to clarify consensus.
+
+---
+
+### Updating the Copyright [TSC#174](https://github.com/nodejs/TSC/issues/174)
+
+See also:
+
+* [WIP] meta: update copyright statements
+  [node#10599](https://github.com/nodejs/node/pull/10599)
+* [WIP] Restore copyright attribution
+  [node#10155](https://github.com/nodejs/node/pull/10155)
+
+@jasnell has formally asked for the Foundation’s legal committee to review and
+provide feedback, response still pending.
+
+Everyone should review proposed final revised form (in #10599).
+
+Even initial reversion (#10155) requires review from legal committee.
+
+Call for a vote (once feedback is received) to clarify consensus.
+
+---
+
+### Growing the group of releasers [CTC#48](https://github.com/nodejs/CTC/issues/48)
+
+Doesn’t require CTC membership, but requires CTC signoff to grant release
+permissions.
+
+Should shadow current releasers first to ensure they understand process.
+
+Grant permissions after initial shadow.
+
+Need to coordinate with Build WG.
+
+Evan Lucas called for a vote to invite @gibfahn and @italocasas to become
+releasers and grant all permissions required. Requires that their public GPG key
+be promoted in release system. Also assumes completion of initial shadow.
+
+Partial vote completed and recorded in issue.
+
+---
+
+### Rewrite 002 - esm [node-eps#39](https://github.com/nodejs/node-eps/pull/39)
+
+Talking points: <https://gist.github.com/bmeck/e11d5618ccfa5c7b57f3cae066c54c2e>
+
+Current proposal ([node-eps#39](https://github.com/nodejs/node-eps/pull/39)) was
+made after July’s TC39 meeting. After September TC39 meeting and recent talks
+with V8, we know what needs to change in next version of
+[node-eps#002-es6-modules.md](https://github.com/nodejs/node-eps/blob/master/002-es6-modules.md)
+on how Node.js will handle ES modules.
+
+In September we asked for property delegation on access of imported variables
+from CommonJS (CJS) modules. V8 team has misgivings on this because it would
+require pessimistically guarding against all potential side effects for
+accessing all imported variables, including in ESM to ESM calls. For example,
+would mean none of those can be inlined.
+
+Therefore cannot import named variables from CommonJS (CJS) modules, e.g.
+`import {readfile} from “fs”`.
+
+So CommonJS modules will provide only a namespace export with `default` and all
```

Comment:
This would break most code written in TypeScript:
Please see:
https://github.com/Microsoft/TypeScript/issues/8687
https://github.com/Microsoft/TypeScript/issues/7398
https://github.com/systemjs/systemjs/issues/1587